.changeset/wise-parents-rule.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": minor
+---
+
+Added: `report`/`code` properties to Node.js API returned object and deprecate `output` property instead
```

Comment:
Suggestion: I think separating the "added" and "deprecated" actions could make it easier for users to notice that a deprecation happened as they scan the changelog. What are your thoughts?

Reply:
I wonder if you can add 2 lines/entries or if you need 2 separate markdown files.

Reply:
Since 2 lines per file are unsupported, I've split the entry into two files with 24eee76c9.